Auth_phpbb.php

```diff
@@ -337,8 +416,7 @@ public function authenticate($username, $password)
         // Connect to the database.
         $fresMySQLConnection = $this->connect();
 
-        $username = $this->utf8($username); // Convert to UTF8
-        $username = $fresMySQLConnection->escape_string($username);
```

Comment:
Because bind_param does it for you.  I anticipated this question, so included "// bind_param escapes the string" in nearby code comments.  PHP was revised the way it was to to improve inherent security by removing all responsibility for manually and tediously escaping strings.